sops/__init__.py

```diff
@@ -583,6 +597,20 @@ def parse_pgp_fp(tree, pgp_fps):
     return tree, has_at_least_one_method
 
 
+def parse_kms_context(tree, kms_context):
+    context = dict()
+    for item in kms_context.split(','):
+        item = item.replace(" ", "")
+        valuepos = item.find(":")
+        if valuepos > 0:
+            context[item[:valuepos]] = item[valuepos + 1:]
+
+    for entry in tree['sops']['kms']:
+        entry["context"] = context
```

Comment:
post-merge remark here:
the method parse_kms_arn(), which execute just before parse_kms_context(),
```
def parse_kms_arn(tree, kms_arns):
    """Take a string that contains one or more KMS ARNs, possibly with roles,
       and transform them it into KMS entries of the sops tree
    """
    has_at_least_one_method = False
    tree['sops']['kms'] = list()
```
will guarantee tree['sops']['kms'] exists and completely new.